client/app/error/error.scss

```diff
@@ -0,0 +1,4 @@
+manager-dedicated-error {
```

Comment:
Wouldn't be covered by `@ovh-ux/manager-error-page` component?

If yes then it would be great to add a comment to explicitly said that we can remove it
as soon as the new module `@ovh-ux/manager-error-page` will be used.

Is that make sense for you?

Reply:
@antleblanc `manager-error-page` is private and will replace only in the `manager` repository
(PR taking care of it is over here: https://github.com/ovh-ux/manager/pull/1266)